int8 request
---                         # Demarcation
float32[] joint_angles    
